db/migrate/20150204174650_add_admin_editable_to_price_group.rb

```diff
@@ -0,0 +1,13 @@
+class AddAdminEditableToPriceGroup < ActiveRecord::Migration
+  def up
+    add_column :price_groups, :admin_editable, :boolean, after: :is_internal, null: false, default: true
+
+    (PriceGroup.globals - PriceGroup.cancer_center).each do |price_group|
```

Comment:
We should probably update the seed file as well for when someone sets up a new environment, base and external are set to false.